components/src/md3/ChannelValue/ChannelValue.tsx

```diff
@@ -0,0 +1,191 @@
+        return color;
+    }, [color, theme]);
+
+    const getIcon = useCallback(() => {
+        if (icon) {
+            return (
+                <View style={[{ marginRight: Math.round(fontSize / 3) }]}>
+                    <Icon source={icon} size={iconSize || fontSize} color={iconColor || getColor()} />
+                </View>
+            );
+        }
+    }, [icon, fontSize, getColor, iconColor, iconSize]);
+
+    const getUnits = useCallback(
+        (spacerLocation: 'before' | 'after'): JSX.Element | undefined => {
+            if (units) {
+                return (
+                    <>
+                        {((spacerLocation === 'before' && unitSpace === 'show') ||
+                            (spacerLocation === 'before' &&
+                                unitSpace === 'auto' &&
+                                !suffixUnitWhitelist.includes(units))) && <Spacer flex={0} width={fontSize / 4} />}
+                        <Text
+                            variant={'bodyMedium'}
+                            // font={'light'}
+                            // fontSize={fontSize}
+                            style={[
+                                {
+                                    color: getColor(),
+                                    fontSize: fontSize,
+                                    fontWeight: '300',
+                                    fontFamily: 'OpenSans-Regular',
+                                },
+                                styles.units,
+                            ]}
+                        >
+                            {units}
+                        </Text>
+                        {((spacerLocation === 'after' && unitSpace === 'show') ||
+                            (spacerLocation === 'after' &&
+                                unitSpace === 'auto' &&
+                                !prefixUnitWhitelist.includes(units))) && <Spacer flex={0} width={fontSize / 4} />}
+                    </>
+                );
+            }
+        },
+        [units, fontSize, getColor, styles, unitSpace]
+    );
+
+    const prefixUnits = useCallback((): JSX.Element | undefined => {
+        if ((!I18nManager.isRTL && prefix) || (I18nManager.isRTL && !prefix)) {
+            return getUnits('after');
+        }
+    }, [prefix, getUnits]);
+
+    const suffixUnits = useCallback((): JSX.Element | undefined => {
+        if ((I18nManager.isRTL && prefix) || (!I18nManager.isRTL && !prefix)) {
+            return getUnits('before');
+        }
+    }, [prefix, getUnits]);
+
+    return (
+        <View style={[defaultStyles.root, styles.root, style]} {...viewProps}>
+            {getIcon()}
+            <Text
+                variant={'bodyMedium'}
+                numberOfLines={1}
+                ellipsizeMode={'tail'}
+                testID={'text-wrapper'}
+                // fontSize={fontSize}
+                style={[{ color: getColor(), fontSize: fontSize, fontFamily: 'OpenSans-Regular' }]}
```

Comment:
Is this a temporary solution? are we going to update `bodyMedium`  in the theme file?

Reply:
Yes, the font-family will be coming from the variant. I only put the channelvalue in here as a way to test the Icon utility. There is another story to update the ChannelValue for MD3